src/garage/torch/_functions.py

```diff
@@ -1,261 +1,157 @@
-"""Utility functions for PyTorch algorithms.
```

Comment:
this file seems totally different -- am i missing something?

Reply:
It's updated. It was a mistake.